### ISSUE TITLE
is a5.4 being worked on?

### ISSUE THREAD

**Issue by radmanplays** [EaglerReborn/support#11]
@ZXMushroom63

**Comment by ZXMushroom63** [comment]
Not really. I'm waiting for a successful DMCA counternotice. I know that the excuses in the DMCA takedowns are pretty dumb, but I still don't want to work on a potentially illegal project.

**Comment by radmanplays** [comment]
@OtterCodes102 DMCA counter notice when?

**Comment by leahanderson1** [comment]
> Not really. I'm waiting for a successful DMCA counternotice. I know that the excuses in the DMCA takedowns are pretty dumb, but I still don't want to work on a potentially illegal project.

I think it would be cool if you made the PluginAPI standalone so you can easily add it to any project.

**Comment by radmanplays** [comment]
> > Not really. I'm waiting for a successful DMCA counternotice. I know that the excuses in the DMCA takedowns are pretty dumb, but I still don't want to work on a potentially illegal project.
> 
> I think it would be cool if you made the PluginAPI standalone so you can easily add it to any project.

like a bookmarklet? or a .java file that can be added to any eaglercraft 1.8.8 client before its compiled?

**Comment by leahanderson1** [comment]
more like multiple files but

**Comment by leahanderson1** [comment]
> @OtterCodes102 DMCA counter notice when?

when i have the time to go to court

**Comment by radmanplays** [comment]
> > @OtterCodes102 DMCA counter notice when?
> 
> when i have the time to go to court

what about self hosted gitlab?

**Comment by leahanderson1** [comment]
that could work
